<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialMenu</class>
 <widget class="QWidget" name="InitialMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InitialMenu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/img/ImagesForCapoo/WindowIcon.png</normaloff>:/img/ImagesForCapoo/WindowIcon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">InitialMenu
{
background-image: url(:/img/ImagesForCapoo/MenuBackGround.png);
}
</string>
  </property>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/Exit.jpg</normaloff>:/img/ImagesForCapoo/Exit.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadSaveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadSave</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/SaveAsArchive.png</normaloff>:/img/ImagesForCapoo/SaveAsArchive.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="NewGameButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>NewGame</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/GameStart.png</normaloff>:/img/ImagesForCapoo/GameStart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ShopButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Shop</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/shop.png</normaloff>:/img/ImagesForCapoo/shop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
